Automattic/wp-calypso
# Gutenboarding: Move Ibis FSE and Reynolds FSE design screenshots to the right folder

#### Changes proposed in this Pull Request

* Move the Ibis FSE and Reynolds FSE design screenshots to the right folder, after they were left behind by https://github.com/Automattic/wp-calypso/pull/44932 🙂 

#### Testing instructions

* `yarn start` Calypso.
* Start the FSE Gutenboarding experiment: `http://calypso.localhost:3000/new?flags=gutenboarding/site-editor`.
* In the second step, make sure all 3 designs show their screenshots.
